client/plots/matrix.controls.js

```diff
@@ -383,20 +383,6 @@ export class MatrixControls {
 					// 	settingsKey: 'geneVariantCountSamplesSkipMclass',
 					// 	processInput: tw => {},
 					// },
-					{
```

Comment:
@congyu-lu these inputs are not moved elsewhere? I think they should be, maybe in the layout. In some cases, the labels are really long and affects the offsets too much

Reply:
They are also in the _Variables_

Reply:
the two in Variables are the same as the two we deleted here: _termGrpLabelMaxChars_ and _rowlabelmaxchars_, _collabelmaxchars_ and _sampleGrpLabelMaxChars_ are inside of Samples